hpi/cloud/food/v1test/restaurant.proto

```diff
@@ -0,0 +1,50 @@
+syntax = "proto3";
+
+package hpi.cloud.news.v1test;
+
+import "google/protobuf/timestamp.proto";
+
+// A restaurant that serves food.
+message Restaurant {
+  // Required, output only. The unique ID of this restaurant.
+  string id = 1;
+
+  // Required. The name of this restaurant.
+  string name = 2;
```

Comment:
Don't call this name as name is usually associated with different data: https://cloud.google.com/apis/design/standard_fields